### DISCUSSION TITLE
[Alert&report] How should the we check the email validity?

### DISCUSSION THREAD

**Discussion by 1nF0rmed** [apache/superset#18325] [Q&A / Help]
The TODO mentions that we need to check the validity of an email:

https://github.com/apache/superset/blob/ea49aa3d2d31d496531f1142fa6a1504b1fe997a/superset/reports/schemas.py#L108-L110

Does this mean the check will be for general email format validity(`someone@service.com`) or should it check if the email exists?

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by junlincc** [comment]
@dpgaspar 🙏
